lale/lib/lale/join.py

```diff
@@ -61,31 +61,31 @@ def _get_join_info(cls, expr_to_parse):
             left_name = expr_to_parse.left.value.attr
         else:
             raise ValueError(
-                "Expression type not supported! Formats supported: it.tbl_name.col_name or it['tbl_name'].col_name"
+                "ERROR: Expression type not supported! Formats supported: it.table_name.column_name or it['table_name'].column_name"
```

Comment:
ok, but would be better to include the problematic expr (stringified) in the formatted string.  Similarly for the others below.

Reply:
Discussed and will incorporate this later if needed.